README.md

```diff
@@ -105,66 +105,61 @@ public class FailoverSampleApp1 {
 
   public static void main(String[] args) throws SQLException {
     // Create a connection.
-    Connection conn = DriverManager.getConnection(CONNECTION_STRING, USERNAME, PASSWORD);
-    // Configure the connection.
-    setInitialSessionState(conn);
-
-    // Do something with method "betterExecuteQuery" using the Cluster-Aware Driver.
-    String select_sql = "SELECT * FROM employees";
-    ResultSet rs = betterExecuteQuery(conn, select_sql);
-    while (rs.next()) {
-      System.out.println(rs.getString("name"));
+    try(Connection conn = DriverManager.getConnection(CONNECTION_STRING, USERNAME, PASSWORD)) {
+      // Configure the connection.
+      setInitialSessionState(conn);
+   
+      // Do something with method "betterExecuteQuery" using the Cluster-Aware Driver.
+      String select_sql = "SELECT * FROM employees";
+      ResultSet rs = betterExecuteQuery(conn, select_sql);
+      while (rs.next()) {
+        System.out.println(rs.getString("name"));
+      }
     }
   }
 
   private static void setInitialSessionState(Connection conn) throws SQLException {
     // Your code here for the initial connection setup.
     Statement stmt1 = conn.createStatement();
-    stmt1.executeUpdate("SET time_zone = +00:00");
+    stmt1.executeUpdate("SET time_zone = \"+00:00\"");
   }
-
+  
   // A better executing query method when autocommit is set as the default value - True.
   private static ResultSet betterExecuteQuery(Connection conn, String query) throws SQLException {
     // Create a boolean flag.
     boolean isSuccess = false;
     // Record the times of re-try.
     int retries = 0;
-
+    
     ResultSet rs = null;
     while (!isSuccess) {
       try {
         Statement stmt = conn.createStatement();
         rs = stmt.executeQuery(query);
```

Comment:
should our sample app be explicitly closing the statement and result set object? Not sure if its important to do that or if just closing the connection object is fine

Reply:
I'd recommend this approach. 
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
You can use similar approach as you do above, try(...) {...}